dist/origin-web-catalogs.css

```diff
@@ -59,7 +59,7 @@
   position: fixed;
   right: 320px;
   top: 60px;
-  z-index: calc(1029);
+  z-index: calc(980);
```

Comment:
It is calc'd by:

z-index: calc(@zindex-navbar-fixed - 50);

in the less file. Idea is to keep it under the navbar but over the main panel.